controllers/storagecluster/prometheus/localcephrules.yaml

```diff
@@ -278,6 +278,17 @@ spec:
       for: 10m
       labels:
         severity: warning
+    - alert: CephMonLowNumber
+      annotations:
+        description: The number of node failure zones available (5) allow to increase the number of Ceph monitors from 3 to 5 in order to improve cluster resilience.
+        message: The current number of Ceph monitors can be increased in order to improve cluster resilience
+        severity_level: info
+        storage_type: ceph
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/CephMonLowNumber.md
+      expr: |
+        (count(ceph_mon_metadata)) < (count(count by (label_failure_domain_beta_kubernetes_io_zone, label_provider_id) (ocs_node_labels))) and (count(count by (label_failure_domain_beta_kubernetes_io_zone, label_provider_id) (ocs_node_labels))) >= 5
+      labels:
+        severity: info
```

Comment:
Better to add new alerts to the mixin file as well: https://github.com/red-hat-storage/ocs-operator/tree/6a493fcccdf21a742e77db3f11be37f3c5e6a968/metrics/mixin/alerts.

Also, if it helps, we can look at removing mixins and just updating these YAML files in the future.

Reply:
@Umanga, I cannot see a good reason to keep "alive" the jsonnet implementation of the alerts. In any case I have started a discussion to have a good view about it it is worth to maintain/increase efforts in them, or just rely in plain yaml format of the prometheus alert rule files.
Apart of that, i understand that alerts should be defined only in one place, if i add the alert in the mixin folder, if anybody generates the alerts we will have this alert duplicated.
If the team decides to use only the "mixins" approach i will create the alert in mixins i a new PR. 
For the moment, and to avoid to introduce delays i will use the current implementation.

Reply:
@umangachapagain WDYT?